cime_config/testdefs/testlist_clm.xml

```diff
@@ -4495,5 +4496,55 @@
     </options>
   </test>
 
+  <test name="SMS_Ld5" grid="f10_f10_mg37" compset="IHistClm60BgcCrop" testmods="clm/default">
+    <machines>
+      <machine name="derecho" compiler="intel" category="interim_restart"/>
+    </machines>
+    <options>
+      <option name="wallclock">00:20:00</option>
```

Comment:
Please add a comment whenever we add a new test, so we have some thoughts about why it's there. Our management of the testlist is very manual and difficult and it helps to have something that lists why a given test is there.

So do this here and each of the new tests.

Reply:
Done (c8fe47813).